src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-form/rbd-form.component.ts

```diff
@@ -217,15 +216,9 @@ export class RbdFormComponent extends CdForm implements OnInit {
     this.rbdForm.get('obj_size').disable();
     this.rbdForm.get('stripingUnit').disable();
     this.rbdForm.get('stripingCount').disable();
-
-    /* RBD Image Format v1 */
-    this.rbdImage.subscribe((image: RbdModel) => {
-      if (image.image_format === RBDImageFormat.V1) {
```

Comment:
v1 format doesn't support any features at all.  Nothing can be enabled or disabled.

Reply:
Yes, it was correct. No need to change anything here.